okapi/services/fieldnotes/upload/docs.xml

```diff
@@ -0,0 +1,49 @@
+<xml>
+    <brief>Upload Fieldnotes</brief>
+    <issue-id>630</issue-id>
+    <desc>
+        <p>Upload a set of one or more fieldnotes records.</p>
+    </desc>
+    <req name='field_notes'>
+        <p>Fieldnotes consist of one or more records in CSV format. Each
+        record comprises a geocache log consisting of four fields:</p>
+        <ul>
+            <li>Geocache Code</li>
+            <li>Date</li>
+            <li>Log Type</li>
+            <li>Log Text</li>
+        </ul>
+        <p>The first three fields are simple entities, the Log Text field is
```

Comment:
Can you clarify what you mean by "simple entities"?

I suspect "Geocache Code" and "Log Types" are plaintext strings, but the "Date" needs more clarification. There are many ways to format a date. Is it ISO 8601?

Reply:
Simple entities are strings withouit line control characters in it. The date is in the format:
```
2023-11-23T10:17:55Z
```
Don't know at this time what this format is called. PR handles it appropriately, I believe

Reply:
This looks like ISO 8601. Do you always expect the timezone to be UTC?

Reply:
I have seen data from: cgeo, Garmin, Looking4Cache, they all com in zulu time

Reply:
Then you're looking for something like:

The Date field needs to be formatted in a very specific subset of the ISO 8601 format - it needs to be a "2023-11-23T10:17:55Z" string, representing the date and time in UTC timezone.

What will happen when the date and/or cache code is invalid - will the entry be ignored, or will the error be raised?

Reply:
> Then you're looking for something like:
> 
> The Date field needs to be formatted in a very specific subset of the ISO 8601 format - it needs to be a "2023-11-23T10:17:55Z" string, representing the date and time in UTC timezone.

I had actually assumed and still do that this handles date/time appropriately regardles whether it is UTC oder not:

`$date  = date("Y-m-d H:i:s", strtotime($n['date']));`

Of course, if the resulting `$date` variable is not what we expect (how would we test anything other than perhaps null?), then I had intended to discard the record, however, I just notices I haven't handled that condition yet in the PR's code.

> 
> What will happen when the date and/or cache code is invalid - will the entry be ignored, or will the error be raised?

I would opt for discarding the record and not throwing an exception. But an exception would be fine with me too.

Reply:
I suggest to copy the docs and implementation from other methods which already take the date argument in ISO 8601 format. See `when` argument [here](https://www.opencaching.de/okapi/services/logs/submit.html).